<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMap</class>
 <widget class="QWidget" name="GameMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="point_1_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Точка 1</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="point_2_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Точка 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toMenuBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to menu</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="point_3_btn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Точка 3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
